assets/js/18f-guide.js

```diff
@@ -10,9 +10,16 @@ $(function() {
       'https://docs.google.com',
       'https://drive.google.com',
       'https://github.com/18F/Accessibility_Reviews',
+      'https://github.com/18F/blog-drafts',
+      'https://github.com/18F/codereviews',
       'https://github.com/18F/DevOps',
       'https://github.com/18F/handbook',
-      'https://github.com/18F/writing-lab'
+      'https://github.com/18F/Infrastructure',
+      'https://github.com/18F/staffing-and-resources',
+      'https://github.com/18F/team-api.18f.gov',
+      'https://github.com/18F/writing-lab',
```

Comment:
Nothing scientific here...I just looked through the first couple pages of [our private repositories](https://github.com/18f?utf8=%E2%9C%93&query=only%3Aprivate) and added the ones that seemed likely to be linked to from our Pages sites.